lib/src/mapbox_map.dart

```diff
@@ -98,6 +100,10 @@ class MapboxMap extends StatefulWidget {
   /// Biggest impact in android
   final bool dragEnabled;
 
+  final bool logoEnabled;
```

Comment:
similar thing here if you use maps created/hosted by mapbox you are required to show the logo. So add some docs for that as well